app/views/benefits/RemoveCompanyBenefitCheckYourAnswersView.scala.html

```diff
@@ -22,37 +22,50 @@
 @import includes._
 @import includes.forms._
 
-@this(mainTemplate: MainTemplate, form: FormWithCSRF)
+@this(main: MainTempleteNew, form: FormWithCSRF)
 
 @(viewModel: RemoveCompanyBenefitCheckYourAnswersViewModel)(implicit request: Request[_], messages: Messages, user: controllers.auth.AuthedUser, templateRenderer: uk.gov.hmrc.renderer.TemplateRenderer, ec: ExecutionContext)
 
 @header = @{
   ptaHeader(
-    displayBackLink = true,
+    displayBackLink = false,
+    showGdsVersion = true,
     mainHeadingText = Messages("tai.checkYourAnswers.title"),
     preHeadingText = Messages("tai.benefits.ended.journey.preHeader"))
 }
 
-@mainTemplate(
-  title = Messages("tai.checkYourAnswers.title"),
-  authedUser = Some(user),
-  pageHeader = Some(header),
-  articleClasses = None
+@main(
+    title = Messages("tai.checkYourAnswers.title"),
+    pageTitle = Some(Messages("tai.checkYourAnswers.title")),
+    authedUser = Some(user),
+    backLinkContent = Some(Messages("tai.back-link.upper")),
+    backLinkId = "backLink"
 ){
 
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        @header
+    </div>
+</div>
+
+<div class="govuk-grid-row">
```

Comment:
these two are redundant

Reply:
<img width="1113" alt="Screen Shot 2022-07-15 at 14 27 51" src="https://user-images.githubusercontent.com/78593709/179232627-77e87dec-ebf7-4631-a050-f445e0ce753c.png">

Removing them does this :/

Reply:
Takes up the whole screen, whereas adding them aligns them to that of the prototype.

Reply:
i mean u have: 
grid-row
grid-2/3

grid-row
grid-2/3